public/js/main.js

```diff
@@ -658,6 +663,14 @@ function fillDataTable() {
     addButtons();
 }
 
+/**
+ * If one of the property key has a . we replace it by the string 'dot' to avoid an error in Tabulator
+ * @param {string} value 
+ */
+function escapeDotCharacters(value) {
+    return value.replace(/\./g, 'dot');
```

Comment:
I am not sure that replacing  " . " with  "dot" is what we are expecting. 
you can propose it as a solution in the ticket .